AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyResultTransformers.swift

```diff
@@ -67,6 +67,96 @@ class IdentifyResultTransformers {
 
     }
 
+    static func processChild(ids: [String],
+                             blockMap: [String: AWSTextractBlock]) -> String {
+        var keyText = ""
+        for keyId in ids {
+            let keyBlock = blockMap[keyId]
+            guard let keyBlockType = keyBlock?.blockType else {
+                continue
+            }
+            switch keyBlockType {
+            case .word:
+                if let text = keyBlock?.text {
+                    keyText += text + " "
+                }
+            default: break
+            }
+        }
+        return keyText
+    }
+
+    static func processValue(ids: [String],
+                             blockMap: [String: AWSTextractBlock]) -> (String, Bool) {
+        var valueText = ""
+        var valueSelected = false
+
+        // VALUE block has a CHILD list of IDs for the WORD block
+        for valueId in ids {
+            let valueBlock = blockMap[valueId]
+            guard let valueRelatioins = valueBlock?.relationships else {
+                continue
+            }
+            for valueRelation in valueRelatioins {
+                guard let ids = valueRelation.ids else {
+                    break
+                }
+                for id in ids {
+                    let wordBlock = blockMap[id]
+                    guard let wordValueBlockType = wordBlock?.blockType else {
+                        continue
+                    }
+                    switch wordValueBlockType {
+                    case .word:
+                        if let text = wordBlock?.text {
+                            valueText += text + " "
+                        }
+                    case .selectionElement:
+                        valueSelected = wordBlock?.selectionStatus == .selected ? true : false
```

Comment:
Seems like we should also do something similar here if we find multiple selectionElements for a specific key/value pair.

Also,

this can be:
```
valueSelected = wordBlock?.selectionStatus == .selected
```

Reply:
Actually, I haven't really think about the situation of selectionElement. But I agree with your point here